README.md

```diff
@@ -1,46 +1,103 @@
-# Paroli
+# Paroli on Orange Pi
 
-Streaming mode implementation of the Piper TTS system in C++ with (optional) RK3588 NPU acceleration support. Named after "speaking" in Esperanto.
+Streaming mode implementation of the Piper TTS system in C++ with RK3588/3566 NPU acceleration support. 
 
 ## How to use
 
-Before building, you will need to fulfill the following dependencies
+Assuming you are running Ubuntu/Debian clean on your Orange Pi RK3588/3566 ([Orange Pi 5 series](https://orangepi.vn/tu-khoa-san-pham/opi5series)/ [Orange Pi 3B](https://orangepi.net/product-tag/orange-pi-3b))
 
-* xtensor
-* spdlog
-* libfmt
-* piper-phoenomize
-* onnxruntime (1.14 or 1.15)
-* A C++20 capable compiler
+1. You first need to instal rknpu lib, the fastest way is using Petrolus ezrknn installer
```

Comment:
Hey no!

This README file is for the entire project. which supports CPU and GPU. It is not appropriate to assume users will run on an Rockchip platform. If you want to expand on the section for RK3588/3566. Add a new section